h5c/vic/src/vic-webapp/src/app/create-vch-wizard/networks/networks.component.ts

```diff
@@ -70,6 +70,7 @@ export class NetworksComponent implements OnInit {
         Validators.pattern(ipPattern)
       ]],
       clientNetworkRouting: [{ value: '', disabled: true }, [
+        Validators.required,
```

Comment:
It looks like this may make the field required. I'm not sure that's the correct behavior. It seems like we want to require that either both `client-network-routing` and `client-network-gateway` are supplied or both are blank.

Reply:
Zach, for the client-network-gateway is it also set for the default route which vCH assigned IP range belongs to? or just for the optional IP range?

Reply:
<img width="940" alt="screen shot 2018-08-31 at 9 48 30 am" src="https://user-images.githubusercontent.com/14900967/44888328-1c884580-ad03-11e8-9f04-5ccb0683e48e.png">
I took a further look at the previous version, I don't think we even need this change. as this change performs the similar functionality as before.